agent/http_client_5/src/test/java/com/intuit/tank/httpclient5/MockLogUtil.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * 
+ */
+package com.intuit.tank.httpclient5;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.intuit.tank.http.TankHttpLogger;
+import com.intuit.tank.logging.LogEventType;
+import com.intuit.tank.logging.LoggingProfile;
+import com.intuit.tank.vm.settings.AgentConfig;
+import com.intuit.tank.vm.settings.TankConfig;
+
+/**
+ * @author denisa
```

Comment:
remove comments... IMHO I'd also prefer never adding @author tags as well. it's extraneous lines of code that feel possessive and provide no real value.